### ISSUE TITLE
Talkback 3.1.4-Beta4 hangs on certin web content controls.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#137]
```
What steps will reproduce the problem?
1. Perform a search from the standard OS search feature
2. Touch or swipe to the web content window containing the results.
3. Talk back then becomes unresponsive until another control gains focus.  
Cannot read search results at all.

What is the expected output? What do you see instead?
Expect to navigate web content control.  Cannot do so, as Talkback is hanging.

What version of the product are you using? On what operating system?
Talkback 3.1.4-Beta4 on Jellybean with a Nexus 7.

Please provide any additional information below.

```

Original issue reported on code.google.com by `TonyHSpe...@gmail.com` on 17 Sep 2012 at 8:50

**Comment by GoogleCodeExporter** [comment]
Original comment by `alanv@google.com` on 20 Sep 2012 at 10:50
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Issue 141 has been merged into this issue.
```

Original comment by `caseybur...@google.com` on 10 Oct 2012 at 6:24

**Comment by GoogleCodeExporter** [comment]
```
Fixed in TalkBack 3.2.1_b1.
```

Original comment by `caseybur...@google.com` on 20 Nov 2012 at 7:29
- Changed state: **Fixed**
